closed/OpenJ9.gmk

```diff
@@ -408,6 +408,9 @@ endif
 ifneq (,$(OPENJ9_DEVELOPER_DIR))
   CUSTOM_COMPILER_ENV_VARS += DEVELOPER_DIR="$(OPENJ9_DEVELOPER_DIR)"
 endif
+ifeq (true,$(OPENJ9_ENABLE_JITSERVER))
+  CUSTOM_COMPILER_ENV_VARS += JITSERVER_SUPPORT=1
+endif
```

Comment:
I'm not thrilled about the use of environment variables to communicate configuration options, but in the cmake case, I believe this is redundant.

Reply:
`JITSERVER_SUPPORT` flag will eventually be removed once `JITServer` is built by default.
cmake will check `JITSERVER_SUPPORT` as a variable definition.

Reply:
Please arrange to set `JITSERVER_SUPPORT` only where necessary (i.e. *not* for cmake).